### ISSUE TITLE
istioctl install - Error: generate config: helm render: load chart: component does not exist

### ISSUE THREAD

**Issue by redjules** [istio/istio#53847]
### Is this the right place to submit this?

- [X] This is not a security vulnerability or a crashing bug
- [X] This is not a question about how to use Istio

### Bug Description

Just starting using Istio latest version (1.24.0) on Windows 11. I followed the [Getting Started](https://istio.io/latest/docs/setup/getting-started) page.
When I run `istioctl install -f samples/bookinfo/demo-profile-no-gateways.yaml -y` or any `istioctl install`, I get the usual blue ship and "Error: generate config: helm render: load chart: component does not exist".

The error seems to come from [here](https://github.com/istio/istio/blob/34e975bd814ae44d81eaa7320769f3cf1d36fc42/operator/pkg/helm/helm.go#L155).

**The same command works perfectly on 1.23.3.** 

### Version

```Text
$ istioctl version
Istio is not present in the cluster: no running Istio pods in namespace "istio-system"
client version: 1.24.0
$ kubectl version --short
error: unknown flag: --short
See 'kubectl version --help' for usage.
$ helm version --short
v3.16.1+g5a5449d
```

### Additional Information

No logs

**Comment by howardjohn** [comment]
Thanks for the report. We had an issue like this long ago I recall where we don't load windows paths correctly. I think we likely accidentally reverted this during a refactoring. 

I suspect the change is simply fixing the forward vs backwards slash path names somewhere but I am not sure where

**Comment by howardjohn** [comment]
https://github.com/istio/istio/pull/32519/files was the original fix. Likely pretty easy fix to apply again, just need someone with a windows machine to test it

**Comment by jcarrothers-sap** [comment]
This problem is still happening in 1.24.1, despite the PR listed here appearing to have been also merged to the `release-1.24` branch and included in 1.24.1.

**Comment by MohitGevaria23** [comment]
Facing the same issue in release-1.24.1

**Comment by zirain** [comment]
I tested with `go build ./istioctl/cmd/` and it worked as expected, let me dig more.

**Comment by howardjohn** [comment]
> I tested with `go build ./istioctl/cmd/` and it worked as expected, let me dig more.

I think you need to run `manifest generate` or similar not just build

**Comment by zirain** [comment]
I found out it, it was my fault, a stupid nit.

**Comment by apwickrama** [comment]
I tried today with 1.24.1 but got the same error.

**Comment by zirain** [comment]
@apwickrama sorry, I made a stuip nit in 1.24.1, and 1.24.2 will come out soon.

**Comment by vinaykumar-1-dev** [comment]
I am encountering this error during installation. Has anyone successfully installed without encountering this issue? Additionally, is there an estimated timeline for a fix if this problem is still unresolved?

**Comment by howardjohn** [comment]
@vinaykumar-1-dev the fixed is supposed o be in 1.24.2. Can you verify you are on 1.24.2?

**Comment by adamstoltz** [comment]
I'm seeing this error again while migrating away from a very dated 1.16 installation to 1.25.0 (going to kubernetes 1.32), and trying to use `istioctl install -f ../path-to-yaml/operator.yaml`.
I was super confused because it works fine from within the target directory, but when using a relative path with ../ it throws this exact same error.
